hi_IN.lproj/Panel.strings

```diff
@@ -0,0 +1,487 @@
+"Copy Content" = "प्रतिलिपि सामग्री";
+
+
+"Total" = "कुल";
+"Word count only support OS X later than 10.10." = "शब्द गणना केवल OS X 10.10 का समर्थन है।";
+"Indent Size for Code" = "कोड के लिए इंडेंट आकार";
+"Auto wrap long lines" = "ऑटो लपेटें लंबी लाइनें";
+
+"Sorry, current version of Typora won't print background color.\nSo dark theme is not yet supported for printing. \n\nPlease choose a light theme or export this document to PDF." = "क्षमा करें, Typora का वर्तमान संस्करण पृष्ठभूमि रंग मुद्रित नहीं करेगा। \nSo डार्क थीम अभी तक प्रिंटिंग के लिए समर्थित नहीं है। \n \n कृपया एक प्रकाश थीम चुनें या इस दस्तावेज़ को पीडीएफ में निर्यात करें।";
+"Export to PDF" = "पीडीएफ को निर्यात करें";
+"Pretty indentation" = "सुंदर इंडेंटेशन";
+"Update markdown reference for this image" = "इस छवि के लिए मार्कडाउन संदर्भ अद्यतन करें";
+
+"No special action" = "कोई विशेष कार्रवाई नहीं";
+"Copy image to current folder (./)" = "वर्तमान फ़ोल्डर में छवि कॉपी करें (./)";
+"Copy image to ./assets" = "छवि को कॉपी करें ./assets";
+"Copy image to ./${filename}.assets" = "छवि को कॉपी करें ./${filename}.assets";
+"Apply above rules to local images" = "स्थानीय छवियों के नियमों के ऊपर लागू करें";
+"Apply above rules to online images" = "ऑनलाइन छवियों के लिए नियमों को लागू करें";
+"Allow upload images automatically based on YAML settings" = "अपलोड छवियों को स्वचालित रूप से YAML सेटिंग्स के आधार पर अनुमति दें";
+
+"Learn More" = "और अधिक जानें";
+"Whitespace / Line Break" = "व्हिटस्पेस / लाइन ब्रेक";
+"When Writing" = "लेखन करते समय";
+"Export / Print" = "निर्यात / प्रिंट";
+"Preserve sequential whitespace and single line break" = "अनुक्रमिक व्हाट्सएप और सिंगल लाइन ब्रेक को संरक्षित करें";
+"Ignore sequential whitespace and single line break" = "अनुक्रमिक व्हाट्सएप और सिंगल लाइन ब्रेक को अनदेखा करें";
+
+"Quit" = "छोड़ना";
+"Quit Typora when last window is closed" = "अंतिम विंडो बंद होने पर टाइपोटा को छोड़ दें";
+"On Launch" = "लॉन्च पर";
+"Open new file" = "नई फ़ाइल खोलें";
+"Restore last closed folders" = "अंतिम बंद फ़ोल्डर्स को पुनर्स्थापित करें";
+"Restore last closed files and folders" = "अंतिम बंद फ़ाइलों और फ़ोल्डरों को पुनर्स्थापित करें";
+"Open custom folder" = "खुला कस्टम फ़ोल्डर";
+"Select custom folder" = "कस्टम फ़ोल्डर का चयन करें";
+"This option may be overridden by \"Close windows when quitting an application\"इस विकल्प को ओवरराइड किया जा सकता है \" = "This option may be overridden by \"Close windows when quitting an application\" in System Preferences";
```

Comment:
@plutonium-239 hi, could you check this line?

Reply:
should be
```
"This option may be overridden by \"Close windows when quitting an application\" in System Preferences" = "इस विकल्प को ओवरराइड किया जा सकता है जब सिस्टम वरीयता में \"Close windows when quitting an application\" अक्षम हो।";
```